TortoiseAndHare/README.md

```diff
@@ -1,6 +1,6 @@
-#TortoiseAndHare
-Simulates the classic Tortioise and Hare race.
-The Tortoise moves steadily, taking smaller steps, but rarely stepping backwards.
-The Hare moves sporadically, leaping larger distances, often times backwards.
-The "OUCH!!" message is displayed when the two move to the same space.
-The winner is diplayed when they reach the end of the track.
+## TortoiseAndHare
+* Simulates the classic Tortioise and Hare race.
+* The Tortoise moves steadily, taking smaller steps, but rarely stepping backwards.
+* The Hare moves sporadically, leaping larger distances, often times backwards.
+* The "OUCH!!" message is displayed when the two move to the same space.
+* The winner is diplayed when they reach the end of the track.
```

Comment:
Consider also adding a paragraph to explain the Java features demonstrated in this project.